msupply-foundation/open-msupply
# Don't allow changing backdated date with lines assigned.

Fixes #5153
Fixes #5146

# 👩🏻‍💻 What does this PR do?

Stops users from changing prescription date when lines are assigned.

If they do they'll get a prompt asking if they want to delete lines and continue.

![image](https://github.com/user-attachments/assets/97b7a25d-1e4b-4518-b97a-7a81f93aa9a4)


## 💌 Any notes for the reviewer?









# 🧪 Testing



- Create a prescription
- Add a line
- Try to change the date
- See prompt
- Check Yes, and No both work as expected.

# 📃 Documentation

  1.Backdate Prescription
